core/src/main/web/app/template/index_template.html

```diff
@@ -109,6 +109,12 @@
   <script src="vendor/bower_components/datetimepicker/jquery.datetimepicker.js"></script>
   <!-- endbuild -->
 
+  <!-- htmlbuild:embedremove -->
+  <script>
+    (function(w){ w.rAF = requestFrame('request'); })(window);
```

Comment:
It should be a global because it's filling for a browser API. Most modern browsers have window.requestAnimationFrame, this is supposed to implement it for browser that don't have it.

However, the standard API is requestAnimationFrame, I don't think we should pollute the global namespace with an additional value. Request-frame docs say that we can just use:
    requestFrame('native'); // re/ declares requestAnimationFrame & cancelAnimationFrame
without assigning it to any value. Is there a reason why we're not doing that?